tests/nlu/extractors/test_duckling_entity_extractor.py

```diff
@@ -152,23 +178,27 @@ def test_duckling_entity_extractor(component_builder):
         )
 
         # 1381536182 == 2013/10/12 02:03:02
-        message = Message(data={TEXT: "Let us meet tomorrow."}, time="1381536182")
-        duckling.process(message)
-        entities = message.get("entities")
+        messages = [Message(data={TEXT: "Let us meet tomorrow."}, time=1381536182)]
+        duckling.process(messages)
+
+        assert len(messages) == 1
+        entities = messages[0].get("entities")
         assert len(entities) == 1
         assert entities[0]["text"] == "tomorrow"
         assert entities[0]["value"] == "2013-10-13T00:00:00.000Z"
 
-        # Test dimension filtering includes only specified dimensions
-        _config = RasaNLUModelConfig(
-            {"pipeline": [{"name": "DucklingEntityExtractor"}]}
-        )
-        _config.set_component_attr(
-            0, dimensions=["number"], url="http://localhost:8000"
-        )
-        duckling_number = component_builder.create_component(
-            _config.for_component(0), _config
-        )
+
+def test_duckling_entity_extractor_dimension_filtering(
+    default_model_storage: ModelStorage, default_execution_context: ExecutionContext
+):
+    config = {
+        **DucklingEntityExtractorComponent.get_default_config(),
+        "dimensions": ["number"],
+        "url": "http://localhost:8000",
+    }
+    duckling_number = DucklingEntityExtractorComponent.create(
```

Comment:
maybe worth extracting the component creation to a helper function with `dimensions` as an argument?

Reply:
done